server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -2242,8 +2242,19 @@ private List<File> buildEngine() throws ExtenderException {
         try {
             outputFiles.addAll(buildPods());
 
+            // An easy way to disable building an extension, is if the symbol name is
+            // disabled at the .appmanifest level
+            List<String> excludeSymbols = ExtenderUtil.getStringList(mergedAppContext, "excludeSymbols");
+
+            // Not sure what the best way to do this is, but
+
             List<String> symbols = getSortedKeys(manifestConfigs.keySet());
             for (String extensionSymbol : symbols) {
+                if (excludeSymbols.contains(extensionSymbol)) {
+                    LOGGER.info("Skipping extension {} due to excludeSymbols in app manifest", extensionSymbol);
+                    continue;
+                }
```

Comment:
This helps with disabling an extension, without need to be overly intrusive to the game project (removing dependencies).
Instead, they can use an appmanifest that simply "excludes" an extension.